src/fides/ctl/core/config/__init__.py

```diff
@@ -108,6 +108,45 @@ def censor_config(config: FidesConfig) -> Dict[str, Any]:
     return filtered
 
 
+def build_config(config_dict: Dict[str, Any]) -> FidesConfig:
+    """
+    This function builds and instantiates a FidesConfig.
+
+    It will pull from the provided config_dict when possible for subsections,
+    otherwise instantiating the settings with default values.
+
+    The settings_map will need to be kept up-to-date manually with the
```

Comment:
This'll be hard to remember and maintain, can you do a pytest that checks these stay in sync that would fail helpfully when someone adds to the settings model without updating the build config function?

Reply:
@NevilleS I'm not sure that is necessary. If someone adds a new config section but forgets to add it here, it will not show up in the config at all and presumably the new functionality would also not work.

I'm also not sure right off-hand what an automated way to do that would be, otherwise I would've definitely avoided this manual solution (not a fan of these kinds of solutions, but I documented it at least!) 

Given the rarity with which we add new config subsections and the unlikelihood of causing any errors in its fail-state, I am ok with it as-is, but lmk if you disagree